### ISSUE TITLE
Getting error on SDWebImage 5.12.0 "Value of type 'UIImageView' has no member 'sd_cancelCurrentAnimationImagesLoad'"

### ISSUE THREAD

**Issue by pawan1789** [SDWebImage/SDWebImage#3328]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by pawan1789** [comment]
Platform Name:  iOS
Platform Version: 15.0
SDWebImage Version: 5.12.3
Inegration method: Carthage
Xcode Version: Xcode 13.1

**Comment by pawan1789** [comment]
Hi @dreampiggy , any way to do this? I need to do this Tableviewcell prepareforreuse method.

**Comment by dreampiggy** [comment]
See migration: https://github.com/SDWebImage/SDWebImage/wiki/5.0-Migration-guide
